recipes/iterative-solver/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "iterative-solver" %}
+{% set version = "0.10.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/molpro/iterative-solver/archive/refs/tags//{{ version }}.tar.gz
+  sha256: d1714e8a59047885cf336934b45e5d0ce6643603ed72ca9e8cf9b7ec928d4a81
+
+build:
+  skip: True  # [win]
+  script:
+    - bash python/build.sh
+    - ls -lR ${CONDA_PREFIX}
```

Comment:
I'm not sure what your upstream `python/build.sh` is doing, but it needs to be installing things into `$PREFIX`, and in particular (for things depending on the python version like this package) into `$SP_DIR` -- i.e. the site-packages directory.

Wherever your build script is copying things, it's not where conda expects them to be, and so it says: "no files found 🤷"

PS. This is one of many reasons why using a vanilla `python -m pip install` install setup is a good idea, because this will automatically pick up the right path from our python.

Reply:
Thank you. I now realise that I have been doing `python -m pip install -e` in my script, and need to get rid of `-e` to get the files actually in the environment tree.